features/0023-did-exchange/README.md

```diff
@@ -282,25 +282,29 @@ The exchange response message is used to complete the exchange. This message is
   },
   "did": "B.did@B:A",
   "did_doc~attach": {
-     "base64": "eyJ0eXAiOiJKV1Qi... (bytes omitted)",
-     "jws": {
-        "header": {
-           "kid": "did:key:z6MkmjY8GnV5i9YTDtPETC2uUAW6ejw3nk5mXF5yci5ab7th"
-        },
-        "protected": "eyJhbGciOiJFZERTQSIsImlhdCI6MTU4Mzg4... (bytes omitted)",
-        "signature": "3dZWsuru7QAVFUCtTd0s7uc1peYEijx4eyt5... (bytes omitted)"
+      "@id": "B.did@B:A",
+      "mime-type": "application/json",
+      "data": {
```

Comment:
Note that there are three more examples in the text that need this treatment (add `"@id"`, `"mime-type"` and subsume the rest into value for `"data"` key)